Ports/README.md

```diff
@@ -11,6 +11,14 @@ environment.
 
 A list of all available ports can be found [here](AvailablePorts.md).
 
+## External ports
+
+You can add third party ports by pointing to a local ports directory in the `SERENITY_PORT_DIRS` variable. For example:
```

Comment:
Maybe I'm still not 100% clear on this functionality: we would never add third party port repositories just for _dependencies_, since all dependencies for "official" ports in SerenityOS would need to be included in the monorepo.

So you would only need this if sm64 needed additional dependencies not in the official ports directory but in a 3rd party repository, correct?

Reply:
correct.

Reply:
In that case, the description in README might be misconstrued; could this work?
```
Third party ports might need additional dependencies from another location. In this case, you can point the `SERENITY_PORT_DIRS` variable to a local ports directory. For example:
```

Reply:
That could work very well.